monitors/test/SyntheticPegMonitor.js

```diff
@@ -15,92 +15,204 @@ contract("SyntheticPegMonitor", function(accounts) {
+      assert.equal(spy.callCount, 1); // There should be no new messages sent.
+
+      // Price deviation below the threshold of 20% should send a message.
+      medianizerPriceFeedMock.setCurrentPrice(toBN(toWei("1")));
+      uniswapPriceFeedMock.setCurrentPrice(toBN(toWei("0.7")));
+      await syntheticPegMonitor.checkPriceDeviation();
+      assert.equal(spy.callCount, 2); // There should be one message sent at this point.
+      assert.isTrue(lastSpyLogIncludes(spy, "off peg alert"));
+      assert.isTrue(lastSpyLogIncludes(spy, "0.7")); // uniswap price
+      assert.isTrue(lastSpyLogIncludes(spy, "1.00")); // expected price
+      assert.isTrue(lastSpyLogIncludes(spy, "30.00")); // percentage error
+    });
   });
 
-  it("Correctly emits messages ", async function() {
-    // Zero price deviation should not emit any events.
-    // Inject prices to feeds
-    medianizerPriceFeedMock.setCurrentPrice(toBN(toWei("1")));
-    uniswapPriceFeedMock.setCurrentPrice(toBN(toWei("1")));
-
-    // Check for price deviation from monitor module.
-    await syntheticPegMonitor.checkPriceDeviation();
-    assert.equal(spy.callCount, 0); // There should be no messages sent at this point.
-
-    // Price deviation above the threshold of 20% should send a message.
-    medianizerPriceFeedMock.setCurrentPrice(toBN(toWei("1")));
-    uniswapPriceFeedMock.setCurrentPrice(toBN(toWei("1.25")));
-    await syntheticPegMonitor.checkPriceDeviation();
-    assert.equal(spy.callCount, 1); // There should be one message sent at this point.
-    assert.isTrue(lastSpyLogIncludes(spy, "off peg alert"));
-    assert.isTrue(lastSpyLogIncludes(spy, "1.25")); // uniswap price
-    assert.isTrue(lastSpyLogIncludes(spy, "1.00")); // expected price
-    assert.isTrue(lastSpyLogIncludes(spy, "25.00")); // percentage error
-
-    // Price deviation at the threshold of 20% should send a message.
-    medianizerPriceFeedMock.setCurrentPrice(toBN(toWei("1")));
-    uniswapPriceFeedMock.setCurrentPrice(toBN(toWei("1.2")));
-    await syntheticPegMonitor.checkPriceDeviation();
-    assert.equal(spy.callCount, 1); // There should be no new messages sent.
-
-    // Price deviation below the threshold of 20% should send a message.
-    medianizerPriceFeedMock.setCurrentPrice(toBN(toWei("1")));
-    uniswapPriceFeedMock.setCurrentPrice(toBN(toWei("0.7")));
-    await syntheticPegMonitor.checkPriceDeviation();
-    assert.equal(spy.callCount, 2); // There should be one message sent at this point.
-    assert.isTrue(lastSpyLogIncludes(spy, "off peg alert"));
-    assert.isTrue(lastSpyLogIncludes(spy, "0.7")); // uniswap price
-    assert.isTrue(lastSpyLogIncludes(spy, "1.00")); // expected price
-    assert.isTrue(lastSpyLogIncludes(spy, "30.00")); // percentage error
+  describe("Pricefeed volatility", function() {
+    beforeEach(async function() {
+      // Tested module that uses the two price feeds.
+      syntheticPegMonitorConfig = {
+        volatilityWindow: 3650,
```

Comment:
why is this the tested window duration?

Reply:
60*60+50?

Reply:
It doesn't actually affect the tests. I just wanted to make sure that if the `volatilityWindow > pricefeed.length` it wouldn't break.